tests/test_security.py

```diff
@@ -0,0 +1,80 @@
+import responses

```

Comment:
You don't assert anything in this test file, is this OK?

Reply:
Yes it is. The only things I can assert are:
  - The HTTP Get request have been made successfully  
  - The returned value of the request
However, those are asserted by the responses package